api/config/feature-toggles-config.js

```diff
@@ -17,6 +17,12 @@ export default {
     defaultValue: false,
     tags: ['team-certification'],
   },
+  isSelfAccountDeletionEnabled: {
+    description: 'Used to enable self account deletion',
+    type: 'boolean',
+    defaultValue: true,
+    tags: ['frontend', 'team-acces'],
```

Comment:
Plutôt que le nom d'une équipe (qui est susceptible de changer), mettre le nom d'un contexte DDD : 
```suggestion
    tags: ['frontend', 'privacy'],
```

Reply:
Je me base sur les autres feature toggles qui utilisent les noms d'équipe

Reply:
Bien reçu. J'évoquerai alors le sujet à nouveau lors d'une réunion avec tous les devs.